src/libstd/sys/unix/net.rs

```diff
@@ -66,7 +67,30 @@ impl Socket {
     }
 
     pub fn duplicate(&self) -> io::Result<Socket> {
-        let fd = try!(cvt(unsafe { libc::dup(self.0.raw()) }));
+        use libc::funcs::posix88::fcntl::fcntl;
+        const ZERO: c_int = 0;
```

Comment:
Not sure why I added `MINUS_ONE`, and will remove that, but `ZERO` is kind of required to make sure the type of `0` is `c_int` as this is passed as varargs to `fcntl`.